net/ddns-scripts/Makefile

```diff
@@ -316,6 +316,11 @@ define Package/ddns-scripts/install
 	$(INSTALL_DIR) $(1)/usr/bin
 	$(INSTALL_BIN) ./files/usr/bin/ddns.sh \
 		$(1)/usr/bin/ddns
+
+	$(INSTALL_BIN) ./samples/slaac_sample.sh \
+		$(1)/usr/lib/ddns
+	$(INSTALL_BIN) ./samples/getlocalip_sample.sh \
+		$(1)/usr/lib/ddns
```

Comment:
**Why do we need to ship also samples in our package?** That's not clear for me. It should be described in the commit description.  Please check: https://openwrt.org/submitting-patches#submission_guidelines

Also, you need to bump PKG_RELEASE.

Reply:
Right, sorry about that! I updated the commit message and PKG_RELEASE, thanks!

Reply:
Then another question came to my mind. Why do we really need to include samples in every installation? There should be subpackage for examples, so it does not take more space than it should have. If you want examples/samples (whatever), you should follow the documentation.

Reply:
well, they are samples, but you can also consider them as utils...
maybe a `ddns-scripts-utils` package?

Reply:
ok, I put together a simple `ddns-scripts-utils` package, and managed to build it (`menuconfig` with the packages repo inside the `buildroot` and then `make package/ddns-scripts/compile`, and confirmed it appeared within my `bin` directory).

Reply:
Okay, now it looks good.